### PR TITLE
Add unit test for PublishMetadata, Version command

### PR DESCRIPTION
**Pull request by BurningCN**
## What is the purpose of the change

Add unit test for Ready, PublishMetadata, Version command

### CODE REVIEW

File: dubbo-plugin/dubbo-qos/src/test/java/org/apache/dubbo/qos/command/impl/PublishMetadataTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import org.apache.dubbo.rpc.model.ApplicationModel;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class PublishMetadataTest {
+
+    @BeforeEach
+    public void before() {
+        ApplicationModel.getConfigManager().setApplication(new ApplicationConfig("app"));
```
Comment (AlbumenJ) [inline dubbo-plugin/dubbo-qos/src/test/java/org/apache/dubbo/qos/command/impl/PublishMetadataTest.java:29]: Please reset ApplicationModel after tests

File: dubbo-plugin/dubbo-qos/src/test/java/org/apache/dubbo/qos/command/impl/ReadyTest.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    private Ready ready;
+
+    @BeforeEach
+    public void setUp(){
+        ready = new Ready();
+    }
+
+    @Test
+    public void appReadyTest(){
+        DubboBootstrap.getInstance().setReady(false);
```
Comment (AlbumenJ) [inline dubbo-plugin/dubbo-qos/src/test/java/org/apache/dubbo/qos/command/impl/ReadyTest.java:46]: Please use `DubboBootstrap.reset()` to revocer env after test

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-plugin/dubbo-qos/src/test/java/org/apache/dubbo/qos/command/impl/PublishMetadataTest.java (hunk 1 above)
Comment (BurningCN) [reply dubbo-plugin/dubbo-qos/src/test/java/org/apache/dubbo/qos/command/impl/PublishMetadataTest.java:30]: done

File: dubbo-plugin/dubbo-qos/src/test/java/org/apache/dubbo/qos/command/impl/ReadyTest.java (hunk 2 above)
Comment (BurningCN) [reply dubbo-plugin/dubbo-qos/src/test/java/org/apache/dubbo/qos/command/impl/ReadyTest.java]: done

**Comment by BurningCN** [comment]
ReadyTest reports NPE during check, I temporarily remove this test class

**Pull request merged**